app/src/main/java/org/ole/planet/myplanet/ui/sync/SyncActivity.java

```diff
@@ -249,6 +244,7 @@ public void onSyncFailed(final String s) {
     public void onSyncComplete() {
         DialogUtils.showSnack(findViewById(android.R.id.content), "Sync Completed");
         progressDialog.dismiss();
+        editor.putLong("lastSynced", System.currentTimeMillis()).apply();
```

Comment:
@yuviii  you dont need to save last synced time here , it is already saved in destory method of SyncManager.java as `LastSync`